### PR TITLE
Use Wayland for supported GUI packages

### PR DESCRIPTION
**Pull request by supechicken**
## Executed successfully
`filezilla`
`bluefish`
`handbrake`
`homebank`
`xfce4_terminal`
`xfce4_panel`
`thunar`

## Wayland support still in beta
`brave`
`gimp`

## Test failed
`firefox`

## Failed to install
`gcalculator`

Tested on `x86_64`

### CODE REVIEW

**Comment by uberhacker** [comment]
@supechicken:  This is interesting.  Does this fix for i686 also?  We'll need new binaries and bump the versions for bluefish, filezilla, handbrake and homebank.  Please remove the existing binaries.

**Comment by supechicken** [comment]
> @supechicken: This is interesting. Does this fix for i686 also? We'll need new binaries and bump the versions for bluefish, filezilla, handbrake and homebank. Please remove the existing binaries.

I haven't test it on `i686` yet, I need to find an x86 version of the Cloudready image to test it since I don't have an i686 only chromebook

**Comment by supechicken** [comment]
> @supechicken: This is interesting. Does this fix for i686 also? We'll need new binaries and bump the versions for bluefish, filezilla, handbrake and homebank. Please remove the existing binaries.

Done

**Comment by uberhacker** [comment]
Bump versions?

**Comment by uberhacker** [comment]
Might as well update while we're at it.
https://www.bennewitz.com/bluefish/stable/source/bluefish-2.2.12.tar.bz2
http://homebank.free.fr/public/homebank-5.4.3.tar.gz

File: packages/bluefish.rb [hunk 1]
```diff
@@ -41,6 +30,14 @@ def self.build
 
   def self.install
```
Comment (uberhacker) [inline packages/bluefish.rb:31]: Does
```
system "./configure #{CREW_PREFIX} --disable-maintainer-mode"
```
work?

File: packages/filezilla.rb [hunk 2]
```diff
@@ -8,19 +8,6 @@ class Filezilla < Package
   source_url 'https://download.filezilla-project.org/client/FileZilla_3.51.0_src.tar.bz2'
```
Comment (uberhacker) [inline packages/filezilla.rb:8]: `version '3.51.0-1'`?

File: packages/handbrake.rb [hunk 3]
```diff
@@ -14,16 +14,11 @@ class Handbrake < Package
     depends_on 'jansson'
```
Comment (uberhacker) [inline packages/handbrake.rb:14]: `version '1.3.3-1'`?

File: packages/filezilla.rb (hunk 2 above)
Comment (supechicken) [reply packages/filezilla.rb:8]: I can't find this version in `https://download.filezilla-project.org/client/`

File: packages/bluefish.rb (hunk 1 above)
Comment (supechicken) [reply packages/bluefish.rb]: Done

**Comment by supechicken** [comment]
`libxfce4ui` =>  4.16.0
`libxfce4util` => 4.16.0
`xfce4_terminal` => 0.8.10
`exo` => 4.16.0
`xfce4_panel` => 4.16.0
`bluefish` => 2.2.12
`homebank` => 5.4.3

Tested on `x86_64`

File: packages/filezilla.rb (hunk 2 above)
Comment (uberhacker) [reply packages/filezilla.rb:8]: It's not a real version.  I just gave you an example of how you can bump.

**Comment by uberhacker** [comment]
I would rather you split these updates into separate pull requests.

**Comment by esprout** [comment]
For anyone else wondering how start brave with wayland:

brave --enable-features=UseOzonePlatform --ozone-platform=wayland

**Pull request closed without merging**